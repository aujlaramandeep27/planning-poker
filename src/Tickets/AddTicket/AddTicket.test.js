import {AddTicket} from "./AddTicket";

it('AddTicket renders correctly', () => {
    const props = {
        goToHost: jest.fn()
    }
    expect(AddTicket(props)).toMatchSnapshot();
});