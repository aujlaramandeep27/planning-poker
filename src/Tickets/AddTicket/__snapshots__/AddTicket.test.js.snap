// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddTicket renders correctly 1`] = `
<div
  className="uk-container uk-text-center@m uk-margin-top uk-margin-bottom"
>
  <h1
    className="uk-heading-small uk-heading-divider"
  >
    Add Ticket
  </h1>
  <a
    href="#/host"
  >
    <span
      className="uk-position-large uk-position-top-left"
      uk-icon="icon: arrow-left; ratio: 3"
    />
  </a>
  <div
    className="uk-align-center uk-width-1-2@m"
  >
    <form
      onSubmit={[Function]}
    >
      <div
        className="uk-margin"
      >
        <div
          className="uk-inline uk-width-1-2"
        >
          <span
            className="uk-form-icon"
            uk-icon="icon: user"
          />
          <input
            className="uk-input"
            name="ticketInputName"
            placeholder="Name"
            type="text"
          />
        </div>
        <button
          className="uk-button uk-button-primary uk-width-1-4"
        >
          ADD
        </button>
      </div>
    </form>
  </div>
</div>
`;
